OpenRA.Mods.Common/Traits/DeliversCash.cs

```diff
@@ -29,6 +29,9 @@ class DeliversCashInfo : ITraitInfo
 		[Desc("Identifier checked against AcceptsDeliveredCash.ValidTypes. Only needed if the latter is not empty.")]
 		public readonly string Type = null;
 
+		[Desc("Cursor to show when hovering over a valid actor to deliver cash to.")]
+		public readonly string Cursor = "enter";
```

Comment:
This will conflict with https://github.com/OpenRA/OpenRA/pull/18110 so I suggest to just remove it already.